src/app/[activityId]/_components/MobileReserveModal.tsx

```diff
@@ -0,0 +1,243 @@
+  const [peopleCount, setPeopleCount] = useState(1);
+  const [isLoading, setIsLoading] = useState(false);
+
+  const year = String(currentDate.getFullYear());
+  const month = String(currentDate.getMonth() + 1).padStart(2, '0');
+  const dates = getCalendarDates(currentDate);
+
+  const { data: detail } = useActivityDetail(activityId);
+  const { data: schedules = [] } = useAvailableSchedule(
+    activityId,
+    year,
+    month,
+  );
+
+  const price = detail?.price ?? null;
+  const totalPrice = price ? price * peopleCount : null;
+
+  const timeOptions = selectedDate
+    ? (schedules.find((s) => s.date === selectedDate)?.times ?? [])
+    : [];
+
+  const handleReserve = async () => {
+    if (!selectedTimeId) return;
+    try {
+      setIsLoading(true);
+      await activitiesApi.postReservation(activityId, {
+        scheduleId: selectedTimeId,
+        headCount: peopleCount,
+      });
+      onReserved();
+      setSelectedDate(null);
+      setSelectedTimeId(null);
+      setPeopleCount(1);
+    } catch (error) {
+      alert('예약에 실패했습니다.');
+      console.error(error);
+    } finally {
+      setIsLoading(false);
+    }
+  };
+
+  const getMonthNameEnglish = (date: Date): string =>
+    new Intl.DateTimeFormat('en-US', { year: 'numeric', month: 'long' }).format(
+      date,
+    );
+
+  useEffect(() => {
+    const originalStyle = window.getComputedStyle(document.body).overflow;
+    document.body.style.overflow = 'hidden';
+
+    return () => {
+      document.body.style.overflow = originalStyle;
+    };
+  }, []);
+
+  return (
+    <div className='fixed inset-0 z-50 flex items-end justify-center bg-black/40'>
+      <div
+        ref={modalRef}
+        className='h-[80vh] w-full overflow-y-auto rounded-t-2xl bg-white p-20'
+      >
+        {/* 상단: 월 표시 */}
+        <div className='mb-20 flex justify-between'>
+          <p className='txt-16_B'>{getMonthNameEnglish(currentDate)}</p>
+          <div className='flex gap-10'>
+            <button
+              onClick={() =>
+                setCurrentDate(
+                  new Date(
+                    currentDate.getFullYear(),
+                    currentDate.getMonth() - 1,
+                    1,
+                  ),
+                )
+              }
+            >
+              <Icon className='h-24 w-24' icon='TriangleLeft' />
+            </button>
+            <button
+              onClick={() =>
+                setCurrentDate(
+                  new Date(
+                    currentDate.getFullYear(),
+                    currentDate.getMonth() + 1,
+                    1,
+                  ),
+                )
+              }
+            >
+              <Icon className='h-24 w-24' icon='TriangleRight' />
+            </button>
+          </div>
+        </div>
+
+        {/* 캘린더 */}
+        <div className='grid grid-cols-7 items-center gap-10 text-center'>
+          {['Sun', 'Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat'].map((day) => (
```

Comment:
```suggestion
          {['S', 'M', 'T', 'W', 'T', 'F', 'S'].map((day) => (
```
💬 디자인과 맞춰주시는 게 좋지 않을까? 싶네요~